homeassistant/components/gpsd/strings.json

```diff
@@ -15,5 +15,26 @@
       "already_configured": "[%key:common::config_flow::abort::already_configured_service%]",
       "cannot_connect": "[%key:common::config_flow::error::cannot_connect%]"
     }
+  },
+  "entity": {
+    "sensor": {
+      "gpsd": {
+        "state": {
+          "2d_fix": "2D Fix",
+          "3d_fix": "3D Fix"
+        },
+        "state_attributes": {
+          "latitude": { "name": "[%key:common::config_flow::data::latitude%]" },
+          "longitude": {
+            "name": "[%key:common::config_flow::data::longitude%]"
+          },
+          "elevation": { "name": "Elevation" },
+          "gps_time": { "name": "Time" },
+          "speed": { "name": "Speed" },
+          "climb": { "name": "Climb" },
+          "mode": { "name": "Mode" }
+        }
```

Comment:
Will these become separate sensors in the future?

Reply:
Yes. Should I add these afterwards?

Reply:
Maybe reconsider the translation key and the unique id (if it wasn't here before the config flow)

Reply:
Done, thanks

Reply:
Done for both, thanks!